include/swift/SILOptimizer/Utils/CFGOptUtils.h

```diff
@@ -185,8 +191,20 @@ bool mergeBasicBlocks(SILFunction *f);
 ///
 /// *NOTE* This completion may not be unique.
 void completeJointPostDominanceSet(
-    ArrayRef<SILBasicBlock *> userBlocks, ArrayRef<SILBasicBlock *> defBlocks,
-    llvm::SmallVectorImpl<SILBasicBlock *> &completion);
+    SmallPtrSetImpl<SILBasicBlock *> &userBlocks,
+    SmallPtrSetImpl<SILBasicBlock *> &defBlocks,
+    llvm::SmallVectorImpl<SILBasicBlock *> &completion,
+    llvm::SmallVectorImpl<SILBasicBlock *> &postDominators);
```

Comment:
Sure. That makes sense. I will change postdominators to joint post-dominating set. Thanks for pointing that out. 

> In fact, this would be a cleaner API if we removed the completion set and let the caller check membership in userBlocks.

Yes. I will do that.